workflow/scripts/scramble_vcf.py

```diff
@@ -0,0 +1,129 @@
+from datetime import date
```

Comment:
For logging use:
import logging

log = logging.getLogger()

in __main__:
log = snakemake.log_fmt_shell(stdout=False, stderr=True)